src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -821,9 +860,10 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
 
             for (Item1 = 1; Item1 <= state.dataHeatBal->InfiltrationObjects(Item).NumOfZones; ++Item1) {
```

Comment:
scope

Reply:
I will make a scope pass.  HBAM definitely didn't get the attention that HeatRecovery got.